ddl/sequence_test.go

```diff
@@ -255,6 +255,11 @@ func (s *testSequenceSuite) TestSequenceAsDefaultValue(c *C) {
 	s.tk.MustExec("create sequence seq")
 
 	// test the use sequence's nextval as default.
+	s.tk.MustExec("drop table if exists t")
+	s.tk.MustExec("create table t(a int not null default next value for seq key)")
+	s.tk.MustExec("drop table if exists t")
+	s.tk.MustExec("create table t(a int not null default nextval(seq), b int, primary key(a))")
```

Comment:
Could we add other `DefaultIsExpr` tests?

Reply:
here the default nextval(seq) is the case of `col.DefaultIsExpr=true`

Reply:
Yes, but I mean other `DefaultIsExpr` exclude `nextval(seq)`.

Reply:
yes, `DefaultIsExpr` is only used by sequence now